IrisHub/SpaceBar
# Sam/video UI

### Description of Changes
Video UI with minimal styling for local audio / video with ability to toggle audio / video states.

Uses browser's navigator.mediaDevices.getUserMedia API to obtain video and or audio streams upon component mount & upon changes to video or audio state.


<img width="1618" alt="Screen Shot 2022-04-19 at 00 00 01" src="https://user-images.githubusercontent.com/31233283/163944553-b2605da5-f967-4ed3-bbbf-07a1b88c04cb.png">

### Link to Notion ticket
https://www.notion.so/irisapp/Video-player-UI-dbb7addf6d9a4af89df402ede590b472

### Testing Procedure
Tested each toggle combination in browser for expected behavior, such as correctly toggling on / off in the browser UI & user's webcam. Tested mostly in Chrome and somewhat in Safari. More work needs to be done to ensure this works across browsers.
